AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -437,8 +437,8 @@ public class ConstructorIO: CIOSessionManagerDelegate {
      constructorIO.trackBrowseResultsLoaded(filterName: "Category", filterValue: "Snacks", resultCount: 674, resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
      ```
      */
-    public func trackBrowseResultsLoaded(filterName: String, filterValue: String, resultCount: Int, resultID: String? = nil, completionHandler: TrackingCompletionHandler? = nil) {
-        let data = CIOTrackBrowseResultsLoadedData(filterName: filterName, filterValue: filterValue, resultCount: resultCount, resultID: resultID)
+    public func trackBrowseResultsLoaded(filterName: String, filterValue: String, resultCount: Int, customerIDs: [String]? = nil, resultID: String? = nil, completionHandler: TrackingCompletionHandler? = nil) {
```

Comment:
Initially I thought this would be a breaking change, but since we've defined the functions with argument labels - this is actually a backwards compatible change! 🤯 

So people can't call the function like the following:
```
self.constructor.trackBrowseResultsLoaded(groupFilterName, groupFilterValue, resultCount, nil, { response in
        let cioError = response.error as? CIOError
        XCTAssertNil(cioError)
        expectation.fulfill()
})
```

Which would cause an error. They actually have to pass in the arguments with `filterName`, `filterValue`, etc (the argument labels). Pretty sweet!

More detailed write-up in this Slack message: https://constructor.slack.com/archives/C02RSEE9QE4/p1694128363919359